common/src/main/java/com/lightstep/tracer/shared/NoopSpan.java

```diff
@@ -0,0 +1,52 @@
+package com.lightstep.tracer.shared;
+
+import io.opentracing.Span;
+
+// A span which is returned when the tracer is disabled.
+public class NoopSpan implements Span {
```

Comment:
This doesn't need to be public IIUC... package protected (i.e., just remove `public`) should be fine.